tests/acceptance/00_basics/def.json/classes.cf.json

```diff
@@ -2,6 +2,8 @@
  "classes":
  {
   "test_class_29665402e2b4331f10b8d767b512cd916eeb5db9": [ "one", "of", "these", "matches_classmatch", "any" ],
+  "test_class_29665402e2b4331f10b8d767b512cd916eeb5db9_2": [ "cfengine" ],
+  "test_class_29665402e2b4331f10b8d767b512cd916eeb5db9_3": "$(sys.os)",
```

Comment:
Right, `$(sys.os)` doesn't actually match the corresponding class on some OSes. Silly, I know, but that's how it is. I think the `"cfengine"` test is sufficient though. Maybe remove this one?